systemtests/scripts/functions

```diff
@@ -113,18 +113,24 @@ setup_data()
 {
   data_dir="${tmp}/data"
 
+  # gnu tar is required
+  if [ "$(uname -s)" == SunOS ]; then
+    GTAR='gtar'
+  else
+    GTAR='tar'
+  fi
+
```

Comment:
maybe we could do this with `command -v gtar tar` to always use `gtar` if it exists or `tar` otherwise.

We should either have should have a find_command function that wraps this. As an alternative we could export the commands we found using cmake into a shellscript that could be sourced. That way we could have `$GTAR_EXECUTABLE` or something like that automatically set by cmake.